### ISSUE TITLE
Unmet peer dependencies

### ISSUE THREAD

**Issue by UrsusSoft** [creativetimofficial/material-dashboard-pro-angular#115]
Hello!

I'm using Material Dashboard Pro Angular v.1.4.4. When I run "npm install", I'm getting these error:

"ng2-select@1.2.0 requires a peer of @angular/common@^2.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-select@1.2.0 requires a peer of @angular/core@^2.3.0 but none is installed. You must install peer dependencies yourself."

What should I do to resolve that problem?

Thanks.

**Comment by chelaruc** [comment]
@UrsusSoft Thank you for using our product! To fix that, you need to replace your package.json with these lines of code:
```
{
  "name": "md-angular-cli-pro",
  "version": "1.4.4",
  "license": "http://md-pro-angular.creative-tim.com/documentation/tutorial",
  "repository": {
    "type": "git",
    "url": "https://github.com/creativetimofficial/material-dashboard-pro-angular.git"
  },
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@agm/core": "1.0.0-beta.0",
    "@angular/animations": "4.4.4",
    "@angular/cdk": "^2.0.0-beta.12",
    "@angular/common": "4.4.4",
    "@angular/compiler": "4.4.4",
    "@angular/core": "4.4.4",
    "@angular/forms": "4.4.4",
    "@angular/http": "4.4.4",
    "@angular/material": "^2.0.0-beta.12",
    "@angular/platform-browser": "4.4.4",
    "@angular/platform-browser-dynamic": "4.4.4",
    "@angular/platform-server": "4.4.4",
    "@angular/router": "4.4.4",
    "angular2-material-datepicker": "0.5.0",
    "animate.css": "3.5.2",
    "arrive": "2.3.1",
    "bootstrap": "3.3.5",
    "bootstrap-material-design": "0.5.10",
    "bootstrap-notify": "3.1.3",
    "bootstrap-select": "1.12.2",
    "bootstrap-tagsinput": "0.7.1",
    "chartist": "0.9.4",
    "chartist-plugin-zoom": "0.4.0",
    "core-js": "2.4.1",
    "datatables": "1.10.12",
    "datatables.net-bs": "1.10.12",
    "datatables.net-responsive": "2.1.1",
    "eonasdan-bootstrap-datetimepicker": "4.17.47",
    "fullcalendar": "3.4.0",
    "googleapis": "19.0.0",
    "hammerjs": "^2.0.8",
    "jasny-bootstrap": "3.1.3",
    "jquery": "1.12.4",
    "jquery-validation": "1.17.0",
    "jvectormap": "2.0.4",
    "moment": "2.18.1",
    "moment-timezone": "0.4.0",
    "ng2-nouislider": "1.6.1",
    "ng2-select": "1.2.0",
    "ngx-chips": "1.4.6",
    "nouislider": "9.2.0",
    "perfect-scrollbar": "0.8.1",
    "rxjs": "5.4.2",
    "sweetalert2": "6.10.1",
    "twitter-bootstrap-wizard": "1.2.0",
    "uglify-js": "1.3.5",
    "validate": "3.0.1",
    "web-animations-js": "2.2.2",
    "zone.js": "0.8.4"
  },
  "devDependencies": {
    "@angular/cli": "1.4.2",
    "@angular/compiler-cli": "4.4.4",
    "@types/bootstrap": "3.3.32",
    "@types/chartist": "0.9.34",
    "@types/jasmine": "2.5.38",
    "@types/jquery": "1.10.31",
    "@types/node": "6.0.73",
    "codelyzer": "2.0.0",
    "jasmine-core": "2.5.2",
    "jasmine-spec-reporter": "3.2.0",
    "karma": "1.4.1",
    "karma-chrome-launcher": "2.0.0",
    "karma-cli": "1.0.1",
    "karma-coverage-istanbul-reporter": "0.2.0",
    "karma-jasmine": "1.1.0",
    "karma-jasmine-html-reporter": "0.2.2",
    "protractor": "5.1.0",
    "ts-node": "2.0.0",
    "tslint": "4.5.0",
    "typescript": "2.4.2"
  }
}
````
And delete `node_modules` folder and run `npm install` again.

Best,
Ciprian

**Comment by chelaruc** [comment]
fixed in v1.4.5
